code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -11,6 +11,8 @@
 			new /obj/item/device/camera_bug(src)
 			new /obj/item/device/multitool/ai_detect(src)
 			new /obj/item/device/encryptionkey/syndicate(src)
+			new /obj/item/weapon/reagent_containers/syringe/mulligan(src)
+			new /obj/item/weapon/switchblade

```

Comment:
I think you can cram infinite items into a box if you override it cause we have infinite bagslots now, it just wont let you put them back in if you take them out.

Also you spend literally all your TC on the box it needs to give you at least one shitty weapon.